build.gradle

```diff
@@ -153,31 +168,64 @@ dependencies {
         preCompile group: 'com.nomagic', name: 'demomagicdraw', version: '190sp3', classifier: 'MagicDraw_Demo_190_sp3_no_install', ext: 'zip'
         preCompile group: 'com.nomagic', name: 'sysml', version: '190sp3', classifier: 'SysML_Plugin_190_sp3_bundle', ext: 'zip'
         preCompile group: 'com.nomagic', name: 'cst', version: '190sp3', classifier: 'Cameo_Simulation_Toolkit_Plugin_190_sp3', ext: 'zip'
+		preCompile group: 'org.apache.fop',name: 'fop', version: '1.1', classifier: fopNameVersion + '-bin', ext: 'zip'
```

Comment:
Would you mind running this file and ideally others modified in this PR through your IDE's code formatter? If you're using IntelliJ, please see https://www.jetbrains.com/help/idea/reformat-and-rearrange-code.html#reformat_module_directory. Should clean up the minor formatting issues like tabs instead of spaces in this line and will make the diff easier to inspect.

Reply:
I am using eclipse.  Try use lfs functionality but I could not push having .gitattribute to git so I just modify the two lines with/t/t to space for now.